pkg/plugins/runtime/k8s/webhooks/injector/injector.go

```diff
@@ -131,6 +131,15 @@ func (i *KumaInjector) InjectKuma(ctx context.Context, pod *kube_core.Pod) error
 		pod.Annotations[key] = value
 	}
 
+	emptyDir := kube_core.Volume{
```

Comment:
We already have this in the gateway container which means it can be moved into the dataplane factory. I can refactor